File: Week6/mario-more.py
from cs50 import get_int
height = get_int("Height: ")

while(height not in range(1,9)):
    height = int(input("Height: "))

for i in range(1, height + 1):
    for j in range(height, i, -1):
        print(" ",end="")

    for k in range(1, i + 1):
        print("#",end="")

    print("  ",end="")

    for l in range(1, i + 1):
        print("#",end="")

    print("")

File: Week1/mario-more.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    //For each line
    for (int i = 1; i <= height; i++)
    {
        //Print spaces
        for (int j = height; j > i; j--)
        {
            printf("%c", ' ');
        }

        //Print first sequence of #
        for (int k = 1; k <= i; k++)
        {
            printf("%c", '#');
        }
        //Print spaces between the sequences
        printf("  ");

        //Print second sequence of #
        for (int l = 1; l <= i; l++)
        {
            printf("%c", '#');
        }

        printf("\n");
    }
}